app/views/acm/conference/2018.haml

```diff
@@ -0,0 +1,167 @@
+#{meta_tag "summary", :name => "twitter:card"}
+#{meta_tag "@cwruacm", :name => "twitter:site"}
+#{meta_tag "EECS @ CWRU -- Link-State 2018 Conference", :name => "twitter:title"}
+#{meta_tag "The fifth annual student-run tech conference at Case Western Reserve University taking place all day on Saturday October 7th, 2018.", :name => "twitter:description"}
+#{meta_tag "http://acm.case.edu" + "#{image_path 'LinkState175px.png'}", :name => "twitter:image"}
+%aside.right
+  = image_tag 'LinkState175px.png'
+  %h2 Important Info
+  %ul.aside-list
+    %li
+      %span.key Date:
+      .value Saturday, October 6th, 2018
+      %span.key Time:
+      .value 9:00AM - 5:00PM
+      %span.key Location:
+      .value
+        %a{:href => 'http://goo.gl/xsJQRd'} Hovorka Atrium
+        %br
+        Agnar Pytte Science Center
+        %br
+        Case Western Reserve University
+      %span.key Contact:
+      .value
+        #{mail_to "rlc131@case.edu", "Ramon Correa"}
+      %span.key Twitter:
+      .value
+        %a{:href => 'https://twitter.com/cwruacm'} @cwruacm
+      %span.key Hashtag:
+      .value
+        %a{:href => 'https://twitter.com/hashtag/linkstate'} #linkstate
+      %span.key Facebook Event:
+      .value
+        %a{:href => 'https://www.facebook.com/events/466158480392746/'} Link-State 2018
+
+%h1 Link-State 2018
+:markdown
+
+  ACM at Case Western Reserve University is proud to present our *sixth annual*
+  Link-State conference. Every fall, we invite a group of awesome speakers to
+  talk about all sorts of tech-related topics, ranging from programming to
+  robotics to entrepreneurship. We welcome students and professionals to join us
+  on October 7 for a day full of great tech talks, free food, and networking with
+  awesome local tech companies.
+
+  You can learn about the history of the conference on the
+  [about](#{url_for('/acm/conference/about')}) page.  You can find our
+  conference code of conduct [here](#{url_for('/acm/conference/code')}).
+
+%ul.toci
+  %li
+    %a{:href => '#register'} Register
+  %li
+    %a{:href => '#sponsors'} Sponsors
+  %li
+    %a{:href => '#schedule'} Schedule
+  %li
+    %a{:href => '#speakers'} Speakers
+
+%h2 Parking
+:markdown
+
+  If you are driving to Link-State, you can park in lot [S-53](https://goo.gl/maps/kfK4xgBsDgM2), also known as the Veale Parking
+  Garage. We are currently working on obtaining parking vouchers. 
+
+#register
+  %h2 Register
```

Comment:
Would probably be good to have this be "tickets coming soon" and then replace with the latest EventBrite frame later.